website/docusaurus.config.ts

```diff
@@ -69,6 +69,9 @@ const config: Config = {
         theme: {
           customCss: "./src/css/custom.css",
         },
+		gtag: {
+		  trackingID: "GTM-MWH2V47T",
```

Comment:
can you please [enable IP anonymization](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag#configuration).
```suggestion
		  trackingID: "GTM-MWH2V47T",
		  anonymizeIP: true,
```